test/unit/std/charconv_test.cpp

```diff
@@ -220,14 +224,31 @@ TYPED_TEST(integral_from_char_test, hexadicimal_number)
     }
 }
 
-TYPED_TEST(integral_from_char_test, to_char)
+// =============================================================================
+// std::to_chars for integral types
+// =============================================================================
+
+TYPED_TEST(integral_from_char_test, to_chars)
 {
     TypeParam val{120};
     char buffer[10];
 
-        // buffer.clear();
-    [[maybe_unused]] auto res = std::to_chars(&buffer[0], &buffer[0] + sizeof(buffer), val);
+    auto res = std::to_chars(&buffer[0], &buffer[0] + sizeof(buffer), val);
 
+    EXPECT_EQ(res.ptr, &buffer[3]);
+    EXPECT_EQ(res.ec, std::errc{});
```

Comment:
Error codes have an implicit bool conversion that I would use instead.

Reply:
google test macro wants to apply the `operator!` first, since I expect the errc to be zero, and fails because the operand type is std::errc. I wouldn't want to static_cast into bool, I think the EQ is then more expressive?

Reply:
Well then it is ok. If it get‘s to complicated don‘t spent time on that